stubs/setuptools/setuptools/config.pyi

```diff
@@ -1,44 +1,2 @@
-from typing import Any
-
-class StaticModule:
-    def __init__(self, name) -> None: ...
-    def __getattr__(self, attr): ...
-
-def patch_path(path) -> None: ...
 def read_configuration(filepath, find_others: bool = ..., ignore_option_errors: bool = ...): ...
-def configuration_to_dict(handlers): ...
 def parse_configuration(distribution, command_options, ignore_option_errors: bool = ...): ...
-
-class ConfigHandler:
```

Comment:
It seems that `setuptools.config` was turned into a package and at least `ConfigHandler` still exists in `setuptools.config.setupcfg`.

Reply:
Sorry, just got the chance to look at this.

It seems like setuptools.config is deprecated and setuptools.config.setupcfg is "provisional and might be removed in the future".

So maybe it's a good thing that we don't copy this over? Conceivably something people would like the type checker to warn about.

Reply:
https://github.com/pypa/setuptools/blob/4d00d5091e0817f6e539c48f4afc2e69b8ac2591/setuptools/config/__init__.py#L17